views/js/chatControl.js

```diff
@@ -1,39 +1,39 @@
-var chatActiveRoom = "general";
-var chatRefreshTimeMs = 30000;
+let chatActiveRoom = "general";
+const chatRefreshTimeMs = 30000;
 
-var refreshChat = function () {
+const refreshChat = () => {
   document.getElementById("chat_messages").innerHTML = fetchChatHTML();
   setChatScrollToBottom();
 }
 
-var periodicallyRefreshChat = function () {
+const periodicallyRefreshChat = () => {
   refreshChat();
   setInterval(refreshChat, chatRefreshTimeMs);
 }
 
-var setChatScrollToBottom = function () {
-  var element = document.getElementById("chat_messages");
+const setChatScrollToBottom = () => {
+  const element = document.getElementById("chat_messages");
   element.scrollTop = element.scrollHeight;
 }
 
-var fetchChatHTML = function () {
-  var chatList = Array(30).fill({
+const fetchChatHTML = () => {
+  const chatList = Array(30).fill({
     "author": "Bill Gates",
     "text": "This is an example of what a chat message will look like."
   });
-  var html = "";
-  chatList.forEach(function (element) {
+  let html = "";
+  chatList.forEach((element) => {
```

Comment:
Can you use the array/join pattern here, rather than string append?

Reply:
#22 actually deletes this function. This was just used for filler text before that was implemented, so this should go away with the merge.